src/bin/reo.rs

```diff
@@ -139,6 +139,20 @@ pub fn main() {
                 )
                 .arg_required_else_help(true),
         )
+        .subcommand(
+            Command::new("inspect")
+                .setting(AppSettings::ColorNever)
+                .about("Print the universe")
+                .arg(
+                    Arg::new("relf")
+                        .long("relf")
```

Comment:
@Karina5005 if you remove this line, it will be possible to call it like this:

```
$ reo inspect a.relf
```

instead of 

```
$ reo inspect --relf=a.relf
```

I like the first option better.

Reply:
fixed